packages/@magic-sdk/react-native/test/spec/react-native-webview-controller/handleReactNativeWebViewMessage.spec.ts

```diff
@@ -95,3 +95,73 @@ test('Ignores event handlers where `messageHandlers` size is === 0', (done) => {
     done();
   }, 100);
 });
+
+test('Process Typed Array in Solana Payload', (done) => {
+  const viewController = createReactNativeWebViewController('asdf');
+  const onHandlerStub = jest.fn();
+
+  viewController.messageHandlers.add(onHandlerStub);
+
+  viewController.handleReactNativeWebViewMessage({
+    nativeEvent: {
+      url: `asdf/send/?params=${ENCODED_QUERY_PARAMS}`,
+      data: JSON.stringify({
+        msgType: `asdf-${ENCODED_QUERY_PARAMS}`,
+        response: {
+          result: {
+            rawTransaction: {
+              constructor: 'Uint8Array',
+              data: '21,0,0,0',
+              flag: 'MAGIC_PAYLOAD_FLAG_TYPED_ARRAY',
+            },
+          },
+        },
+      }),
+    },
+  } as any);
+
+  setTimeout(() => {
```

Comment:
Hopefully we are using https://jestjs.io/docs/timer-mocks here. Can you confirm?

Reply:
I'm not sure. I was just imitating the other test cases as this is my first unit test in this project.

Based on the context here, this `setTimeout` is only used in the unit test, so I don't think we should mock it.

Reply:
The issue is that unless mocked, each test of this form adds a 100ms to the test run. Not alot but adds up over time. If we mock the setTimeout correctly, then it only adds the time it takes to run the test. Eg, 20 tests would take 10-20ms to run instead of 2000. 

For now, lets just leave it but may be worth creating a cooldown ticket for.

Reply:
Sg